### PR TITLE
portrait hw

### PR DESCRIPTION
**Pull request by AlexZholud**

### CODE REVIEW

File: portrait_hw/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const userBirthYear = Number(prompt ('Enter your year of birth:'));
+let userCity = prompt('Enter the city where you live:');
+let userFavoriteSport = prompt('Enter your favorite kind of sport:');
+
+userCity = userCity.substr(0,1).toUpperCase() + userCity.substr(1).toLowerCase();
+userFavoriteSport = userFavoriteSport.toLowerCase();
+
+let thisDay = new Date();
```
Comment (web-scavenger) [inline portrait_hw/index.js:8]: do you really need this variable

File: portrait_hw/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const basketball = `${likeAChampion}LeBron James?`;
+const beerMile = `${likeAChampion}Me ?:D`;
+
+const cancelBirthYear = 'Your age is a mystery to me :)\n';
+const cancelCity = 'Sorry, i don\'t know where you live, so can\'t visit you with the sweet gifts :(\n';
+const cancelSport = 'Don\'t have a favorite kind of sport? I advise you to learn about the Beer Mile :)';
+
+const acceptBirthYear = `You are ${userAge} years old.\n`;
+const acceptCity = `${liveIn}the beautiful ${userCity} city.\n`;
+const acceptSport = `Your favorite sport is ${userFavoriteSport}. Not bad!:)`;
```
Comment (web-scavenger) [inline portrait_hw/index.js:29]: It's redundant to create ALL messages)

**Review by web-scavenger — APPROVED** [review]

File: portrait_hw/index.js (hunk 2 above)
Comment (AlexZholud) [reply portrait_hw/index.js:29]: This is for starting practice and working with variables , to trying combine variables

File: portrait_hw/index.js (hunk 1 above)
Comment (AlexZholud) [reply portrait_hw/index.js:8]: I thought, if I should take a year, I should take it from the full date, || not? I'll repeat the information about  working with the date

**Pull request merged**